packages/bruno-app/src/components/RequestPane/Script/index.js

```diff
@@ -41,26 +41,28 @@ const Script = ({ item, collection }) => {
   return (
     <StyledWrapper className="w-full flex flex-col">
       <div className="flex-1 mt-2">
-        <div className="mb-1 title text-xs">Pre Request</div>
-        <CodeEditor
+        <h3 className="title text-xs mb-2">Pre Request</h3>
+        <MonacoEditor
           collection={collection}
           value={requestScript || ''}
           theme={storedTheme}
+          height={'25vh'}
           font={get(preferences, 'font.codeFont', 'default')}
-          onEdit={onRequestScriptEdit}
+          onChange={onRequestScriptEdit}
           mode="javascript"
           onRun={onRun}
           onSave={onSave}
         />
       </div>
-      <div className="flex-1 mt-6">
-        <div className="mt-1 mb-1 title text-xs">Post Response</div>
-        <CodeEditor
+      <div className="flex-1 pb-6 mt-2">
+        <h3 className="mb-2 title text-xs">Post Response</h3>
```

Comment:
Changed to h3, but the same `Post response` title is also at least on request Vars tab. So would leave it as it was or change it everywhere to look the same.